src/Lesson10/Task32/Task32.java

```diff
@@ -0,0 +1,39 @@
+package Lesson10.Task32;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+public class Task32 {
+    public static void main(String args[]) {
+        String str = "Java  осень Java   улетают на юг. ";
+        String[] words = str.split(" ");
+        Random rand = new Random();
+        Integer r = rand.nextInt(10);
+        HashMap<Integer, String> hMap = new HashMap<>();
```

Comment:
It would be better to create map, where word will be the key, and word count - value